easybib_vagrant/providers/user.rb

```diff
@@ -0,0 +1,72 @@
+action :create do
+  username = new_resource.username
+  home_dir = new_resource.home_dir
+  composer_token = new_resource.composer_token
+  Chef::Log.info("easybib_vagrant_user: home_dir is #{home_dir}, user is #{username}")
+
+  ['.config/easybib', '.ssh', '.vagrant.d'].each do |path|
+    dir = "#{home_dir}/#{path}"
+    directory dir do
+      mode 0700
+      owner username
+      recursive true
+    end
+  end
+
+  template "#{home_dir}/.ssh/config" do
+    owner username
+    cookbook 'easybib_vagrant'
+    source 'ssh-config-opsworks.erb'
+    mode 0600
+    only_if { is_aws }
+  end
+
+  config = node['easybib_vagrant']['plugin_config']['bib-vagrant'].to_hash
+  config['composer_github_token'] = composer_token
+
+  template "#{home_dir}/.config/easybib/vagrantdefault.yml" do
+    owner username
+    cookbook 'easybib_vagrant'
+    source 'vagrantdefault.yml.erb'
+    variables(
+      :config => config
+    )
+  end
+
+  template "#{home_dir}/.vagrant.d/Vagrantfile" do
+    owner username
+    cookbook 'easybib_vagrant'
+    source 'Vagrantfile.erb'
+    mode 0644
+  end
+
+  node['easybib_vagrant']['plugin_config'].each do |plugin,|
+    bash "Install plugin: #{plugin}" do
```

Comment:
Yes, `bash` vs `sh`, results in not having to use https://github.com/till/easybib-cookbooks/blob/t/vagrant-ci-deployment/easybib_vagrant/recipes/configure.rb#L17-L18